src/reconcile/rec_row.c

```diff
@@ -751,6 +751,13 @@ __wt_rec_row_leaf(
 
         /* Build value cell. */
         if (upd == NULL) {
+            /* Clear the on-disk cell time window if it is obsolete. */
+            if (twp->start_ts != WT_TS_NONE) {
+                __wt_rec_time_window_clear_obsolete(session, twp, r);
+                if (twp->start_ts == WT_TS_NONE)
+                    F_SET(vpack, WT_CELL_UNPACK_TIME_WINDOW_CLEARED);
```

Comment:
Why we need to flag it here?

Reply:
Column store always builds the cell whereas row store doesn't. So based on the flag, it decides whether to rebuild the cell or not? If we clear the timestamps then only we need to rebuild the cell, otherwise not.

Reply:
In particular it's tested about fifteen lines later :-)

Reply:
May be we should be consistent always flag this no matter it's column or row store.

Reply:
Hmm, setting that flag doesn't have a use in a column store, there is no code for it. Maybe I can change the code to set a bool variable and take a decision similar to the flag if that flag is confusing.